eclipse-sw360/sw360
# fix(rest): Fixing the rest test cases

[//]: # (This program and the accompanying materials are made)
[//]: # (available under the terms of the Eclipse Public License 2.0)
[//]: # (which is available at https://www.eclipse.org/legal/epl-2.0/)
[//]: # (SPDX-License-Identifier: EPL-2.0)

> Please provide a summary of your changes here.


Fixed the rest test cases.

> * Which issue is this pull request belonging to and how is it solving it? (*[Refer to issue here](https://github.com/eclipse-sw360/sw360/issues/2510)*)

Issue: 

Ref: #2510

### Suggest Reviewer
> You can suggest reviewers here with an @mention.

### How To Test?
> How should these changes be tested by the reviewer?
   * Build can be tested by CI
   * Need to check the rest documentation 

### Checklist
Must:
